nautobot/core/templates/generic/object_list.html

```diff
@@ -20,14 +37,54 @@
     {% endif %}
 </div>
 <h1>{% block title %}{{ content_type.model_class|meta:"verbose_name_plural"|bettertitle }}{% endblock %}</h1>
+{% if filter_params %}
+<div class="filters-applied">
+    <b>Filters:</b>
+    {% for field, values in filter_params %}
+    <span class="select2 select2-container select2-container--bootstrap display-inline" dir="ltr">
```

Comment:
Still not sure I'm comfortable with using select2 classes for non-select2 elements. I'd rather have a distinct class for each of these new UI elements, even if it means duplicating some CSS.

Reply:
@glennmatthews I can create separate classes clone from select2.